docs/source/api/tracker.md

```diff
@@ -81,45 +81,33 @@ is automatically stopped and won't be rerun.
 
 ### Tracker.autorun and async callbacks
 `Tracker.autorun` can accept an `async` callback function.  
-However, to make the async call reactive, you should wrap your async function in
-a `Tracker.withComputation` call.
+ To preserve reactivity for the reactive variables inside the async callback function, you must use a `Tracker.withComputation` call as described below:
+
+{% apibox "Tracker.withComputation" %}
 
 ```javascript
 Tracker.autorun(async function example1(computation) {
-  let asyncData = await asyncDataFunction();
-  let users =
-    await Tracker.withComputation(computation, () => Meteor.users.find({}).fetch());
-});
-```
-> If you want to get computation in other way you can use `Tracker.currentComputation`
-
-
-{% apibox "Tracker.withComputation" %}
+	// Code before the first await will stay reactive.
+	reactiveVar1.get() // will trigger rerun
 
-In general, the rules to use `Tracker.withComputation` like this:
-1. `async` function *before the first* `await` if just like a sync one.
-2. `async` function *after the first* `await` looses the `Tracker.currentComputation`
-but it can be restored using `Tracker.withComputation`, *but only within the callback*.
+  let links = await LinksCollection.findAsync({}).fetch(); // first async call will stay reactive.
 
-If you have for example:
+	// Code after the first await looses Tracker.currentComputation: no reactivity.
+	reactiveVar2.get() // won't trigger rerun
+ 
+	// You can bring back reactivity with the Tracker.withCompuation wrapper:
+  let users =
+    await Tracker.withComputation(computation, () => Meteor.users.findAsync({}).fetch());
 
-```javascript
-Tracker.autorun(async function (computation) {
-  let asyncData = await someAsyncCall();
-  let links = await LinksCollection.find({}).fetch(); 
-  // code above will not trigger reruns.
+	// Code below will again not be reactive, so you will need another Tracker.withComputation
+	const value = Tracker.withComputation(computation, () => reactiveVar3.get()) // will trigger rerun
 });
 ```
-You can make this example reactive by wrapping the `Meteor.users.find` call in a `Tracker.withComputation` call:
 
-```javascript
-Tracker.autorun(async function (computation) {
-  let asyncData = await someAsyncCall();
-  let users =
-    await Tracker.withComputation(computation, () => Meteor.users.find({}).fetch());
-  // code above will trigger reruns.
-});
-```
+As a rule of thumb, you are fine with wrapping all reactive statements inside a Tracker.withComputation to preserve current computation.
```

Comment:
* Function names should be wrapped with `` ` ``.
* Yes, it is _fine_, but it comes at a performance cost - it should be used only where needed.